docs/getting-started/local.rst

```diff
@@ -75,32 +75,24 @@ the usual ways. It fundamentally provides a CLI command,
 ``snake_case`` is weird, sorry), so you will want to install using a mechanism
 that is conducive to making CLI commands available.
 
-Additionally, this program currently only runs against Python 3.6 or
-Python 3.7, so you will need that installed. (Most Linux distributions ship
-with earlier versions.) Use `pyenv`_ to get Python 3.7 installed in a
-friendly way.
-
 .. code-block:: shell
 
-    # Due to its experimental state, this tool is not published to a
-    # package manager; you should clone it.
-    # (You can pip install it from GitHub, not not if you want to tinker.)
-    git clone https://github.com/googleapis/gapic-generator-python.git
-    cd gapic-generator-python/
+    # Install this package using
+    pip install gapic-generator
 
 	# Install a version of python that is supported by the microgenerator.
```

Comment:
Any particular reason this section is indented one block more?

Reply:
Fixed